src/util.js

```diff
@@ -165,19 +192,21 @@ function deserialize (data) {
 /**
  * Calculate the CID of the binary blob.
  *
- * @param {Object} binaryBlob - Encoded IPLD Node
+ * @param {Uint8Array} binaryBlob - Encoded IPLD Node
  * @param {Object} [userOptions] - Options to create the CID
- * @param {number} [userOptions.cidVersion=1] - CID version number
- * @param {string} [userOptions.hashAlg] - Defaults to the defaultHashAlg of the format
- * @returns {Promise.<CID>}
```

Comment:
does this get in the way of TS checking?

Reply:
Yes - because we require `CID.CIDVersion` and `multihash.HashCode` elsewhere we can't pass any old `number` or `string` in any more.